ArjunaJTS/orbportability/classes/com/arjuna/orbportability/ORBType.java

```diff
@@ -67,16 +67,14 @@ static int getORBEnum(String name)
     public static final int HPORB = 2;
     public static final int VISIBROKER = 3;
     public static final int JAVAIDL = 1;
-	public static final int JACORB = 4;
-    public static final int IBMORB = 5;
+    public static final int IBMORB = 4;
```

Comment:
This is a public interface, I am thinking we should probably keep the same ID number as it would otherwise be a public API change. Are the numbers expected to be contiguous? - if so it might need more consideration.

Reply:
> Are the numbers expected to be contiguous? 

I do not think so.

> This is a public interface, I am thinking we should probably keep the same ID number as it would otherwise be a public API change

It makes sense. I will keep `IBMORB = 5`